source/getting-started/core-concepts.markdown

```diff
@@ -0,0 +1,94 @@
+
+A device can provide multiple entities. For example, a device called “Sam’s Phone” might have entities like location, battery level, and step count. Some devices like lightbulbs only have a single entity, and devices like hubs may have no entities at all.
+
+Some entities are useful for controlling things in your home:
+
+- **Switches** let you check if something is switched on or off?
+- **Sensors** can be a wide range of things that watch for changes in the environment, like motion sensors and temperature sensors
+- **Binary sensors** detect something that is either on or off
+- **[Scripts](https://www.home-assistant.io/docs/scripts/)** are special entities that run custom code when they’re triggered
+
+Other entities are there to respond to Home Assistant:
+
+- **Lights** can turn on and off, and sometimes dim and change color
+- **Covers** are things that can move up and down, like garage doors and smart blinds
+
+Some entities are part of a **domain**, or a built-in category of entities that are managed in a similar way. For example, there is a “lights” domain in Home Assistant.
+
+Entities can be grouped into **areas**, which roughly represent a room in your house. Common areas are “kitchen”, “office”, “basement” and “yard”.
+
+You can reference multiple entities at one time by putting them into a **[group](https://www.home-assistant.io/integrations/group/)**
+
+## Automation
+
+**[Automations](https://www.home-assistant.io/docs/automation/)** let you go father by telling Home Assistant to automatically do things without you having to hit a button yourself. For example, you could write an automation that says “turn on the lights when it gets dark if I’m home”.
+
+An automation has three main parts:
+
+- **[Triggers](https://www.home-assistant.io/docs/automation/trigger/)** - something that starts the automation, like the sun going down or a motion sensor noticing movement
+- **[Conditions](https://www.home-assistant.io/docs/automation/condition/)** - let you decide whether to complete the automation. For example: "only if someone is home".
+- **[Action](https://www.home-assistant.io/docs/automation/action/)** - what the automation does, like "turn on lights"
+
+**[Templates](https://www.home-assistant.io/docs/automation/templating/)** let advanced users use variables and logic in their automations
+
+## Events and state
+
+Entities in Home Assistant are controlled by sending **[events](https://www.home-assistant.io/docs/configuration/events/)**.
+
+Home Assistant remembers the **[state](https://www.home-assistant.io/docs/configuration/state_object/)** of each entity. The state means what *mode* or *condition* the entity is in this moment. For example, a light could have two states: ‘on’ and ‘off’, and a garage door could be in one of two states: ‘open’ or ‘closed’.
```

Comment:
mode & conditions are different terms and things in the Home Assistant world. Let's not put that up the same level as a state.

A state represents the actual physical state a device has in the real world.

Reply:
whoops!

I'll try a different way of defining it, don't want to use the word in its own definition if i can help it though